### ISSUE TITLE
Please check if duplicate

### ISSUE THREAD

**Issue by pcav** [NationalSecurityAgency/qgis-shapetools-plugin#1]
There are other plugins that do similar things. Could you please check and see if there is an opportunity to merge, preferably as Processing subplugins?
I'm sorry to bother, but we are trying hard to reduce redundancy and confusion for the users, and to promote cooperation among plugin developers.
I'm ready to help, just let me know if I can do something.

**Comment by hamiltoncj** [comment]
I would not have written this if there was anything closely related. There are several ellipse generation but they are for either interactively working with them or are very basic in design. Some of them have not had their code updated for a number of years. They may be dead projects and I know some of them could use some help. The Create Shapes is all about the layer data and bulk generating from the data although I have fleshed out the tools to support defaults as well. Ellipse, CA Digitize, MMQGIS and I looked through others - not taking the time right now to go back through them but I have looked at them all. They certainly don't do what we need and I don't think they are the right framework. The other issue is that Shape tools needs to portray the shapes accurately on the earth especially the ellipse and line of bearing.  I have fleshed out the tools to be more than what we need just to make them more useful to the community.

Just let this plugin go through. I have done this due diligence.

That being said I will be sending out an e-mail to the developer group about all the geometry editing tools out there. I think consolidation is in order, but it will take me some time to really go through everything. Additionally, I would like to also address processing algorithms in ArcGIS that are needed in QGIS. I can potentially work on some of them.
